source/intro_rl/offline_rl_en.rst

```diff
@@ -0,0 +1,81 @@
+OOD, also known as distributional shift on training due to the action distribution shift, means that the state-action pair used for learning the Q-function does not exist in the training dataset because the dataset does not contain the action for the given state sampled from the learned policy.
+
+Q&A
+----
+Q1: What are the differences between the offline RL and the imitation learning~(IL)?
+ - Answer： Offline reinforcement learning, henceforth Offline RL, is closely related to imitation learning (IL) in that the latter also learns from a fixed dataset without exploration. However, there are several key differences.
+   
+     - Offline RL algorithms~(so far) have been built on top of standard off-policy Deep Reinforcement Learning (Deep RL) algorithms, which tend to optimize some form of a Bellman equation or TD difference error.
+     - Most IL problems assume an optimal, or at least a high-performing, demonstrator which provides data, whereas Offline RL may have to handle highly suboptimal data.
+     - Most IL problems do not have a reward function. Offline RL considers rewards, which furthermore can be processed after-the-fact and modified.
+     - Some IL problems require the data to be labeled as expert versus non-expert. Offline RL does not make this assumption.
+
+Q2: What is the difference between the off-policy RL and the offline RL?
+ - Answer： The term “off-policy reinforcement learning” is typically used as an umbrella term to denote all reinforcement learning algorithms that can employ datasets of transitions :math:`\mathcal{D}` where the corresponding actions in each transition were collected with any policy other than the current policy :math:`\pi(\mathbf{a} \mid \mathbf{s})`. Q-learning algorithms, actor-critic algorithms that utilize Q-functions, and many model-based reinforcement learning algorithm are off-policy algorithms. However, off-policy algorithms still often employ additional interaction (i.e., online data collection) during the learning process. Therefore, the term “fully off-policy” is sometimes used to indicate that no additional online data collection is performed. Another commonly used term is “batch reinforcement learning”. See the above Figure for a pictorial illustration.
+  
+
+Reference
+----------
+
+1. Offline (Batch) Reinforcement Learning: A Review of Literature and Applications
+2. Levine, Sergey, et al. "Offline reinforcement learning: Tutorial, review, and perspectives on open problems." arXiv preprint arXiv:2005.01643 (2020).
+3. Agarwal, Rishabh, Dale Schuurmans, and Mohammad Norouzi. "An optimistic perspective on offline reinforcement learning." ICML, 2020.
+4. Gulcehre, Caglar, et al. "Rl unplugged: Benchmarks for offline reinforcement learning." Neurips, 2020.
+5. Fu, Justin, et al. "D4rl: Datasets for deep data-driven reinforcement learning." arXiv preprint arXiv:2004.07219 (2020).
```

Comment:
I think this is a very great overview indeed!

To make more perfect and clearer, I guess we need to be specific about the following questions: (Only for suggestions)

1. What is offline RL and why it is needed/becomes popular?
2. Why is offline RL hard (where specifically do we suffer from distributional shift (maybe some diagrams would help, why does online algo suffer less from this?)
3. How do we design offline RL algos to address this distributional shift? (are there any principles we can follow?) maybe mention some algo examples with a clear reference (e.g. IQL,CQL,TD3BC). 
4. How do we evaluate offline RL? (i.e., what indeed do we expect offline RL to do? Maybe we can show it is better than BC class (e.g. BC, DT) in some envs or under some assumptions regarding MDP, but in which way really? I guess compositionality needs to be mentioned here.)
5.  How do we apply offline RL in the real world. As we mentioned in the beginning that offline RL can be applied to large-scale real-world applications, there are indeed some exciting examples or demos that we can refer here to raise the readers interest.
6. We can finally add some extensions to offline RL. For instance, give some reference to model based offline RL and multitask offline RL etc.

I think we have described perfectly for the 1st question. But we may need to be more specific with the second one (better with some diagrams). I guess adding 3rd, 4th, 5th, 6th would make the whole review more professional. 

P.S. I suppose we should be more clear about where we referred those materials in the reference but this is only a suggestion and I think current reference style is already very nice.